config/mbed_config.h

```diff
@@ -19,263 +19,72 @@
+
+// CORTEX Macros
+#define ARM_MATH_CM4
+#define __CORTEX_M4
+#define __CMSIS_RTOS
+#define __MBED_CMSIS_RTOS_CM
+#define __FPU_PRESENT 1
+
+// ST Macros
+#define USE_HAL_DRIVER
+#define USE_FULL_LL_DRIVER
+
+// MBED configuration parameters TODO(wmmc88): update based on mbed_lib.jsons
+// https://os.mbed.com/docs/mbed-os/v6.0/program-setup/advanced-configuration.html and remove any unused params
+#define LPTICKER_DELAY_TICKS                                 1     // set by target:FAMILY_STM32
+#define MBED_CONF_DRIVERS_UART_SERIAL_RXBUF_SIZE             256   // set by library:drivers
+#define MBED_CONF_DRIVERS_UART_SERIAL_TXBUF_SIZE             256   // set by library:drivers
+#define MBED_CONF_EVENTS_PRESENT                             1     // set by library:events
+#define MBED_CONF_EVENTS_SHARED_DISPATCH_FROM_APPLICATION    0     // set by library:events
+#define MBED_CONF_EVENTS_SHARED_EVENTSIZE                    768   // set by library:events
+#define MBED_CONF_EVENTS_SHARED_HIGHPRIO_EVENTSIZE           256   // set by library:events
+#define MBED_CONF_EVENTS_SHARED_HIGHPRIO_STACKSIZE           1024  // set by library:events
+#define MBED_CONF_EVENTS_SHARED_STACKSIZE                    2048  // set by library:events
+#define MBED_CONF_EVENTS_USE_LOWPOWER_TIMER_TICKER           0     // set by library:events
+#define MBED_CONF_PLATFORM_CRASH_CAPTURE_ENABLED             0     // set by library:platform
+#define MBED_CONF_PLATFORM_CTHUNK_COUNT_MAX                  8     // set by library:platform
+#define MBED_CONF_PLATFORM_DEFAULT_SERIAL_BAUD_RATE          ROVER_DEFAULT_SERIAL_BAUD_RATE  // set by library:platform
+#define MBED_CONF_PLATFORM_ERROR_ALL_THREADS_INFO            0                               // set by library:platform
+#define MBED_CONF_PLATFORM_ERROR_FILENAME_CAPTURE_ENABLED    0                               // set by library:platform
+#define MBED_CONF_PLATFORM_ERROR_HIST_ENABLED                0                               // set by library:platform
+#define MBED_CONF_PLATFORM_ERROR_HIST_SIZE                   4                               // set by library:platform
+#define MBED_CONF_PLATFORM_ERROR_REBOOT_MAX                  1                               // set by library:platform
+#define MBED_CONF_PLATFORM_FATAL_ERROR_AUTO_REBOOT_ENABLED   0                               // set by library:platform
+#define MBED_CONF_PLATFORM_FORCE_NON_COPYABLE_ERROR          0                               // set by library:platform
+#define MBED_CONF_PLATFORM_MAX_ERROR_FILENAME_LEN            16                              // set by library:platform
+#define MBED_CONF_PLATFORM_POLL_USE_LOWPOWER_TIMER           0                               // set by library:platform
+#define MBED_CONF_PLATFORM_STDIO_BAUD_RATE                   ROVER_DEFAULT_SERIAL_BAUD_RATE  // set by library:platform
+#define MBED_CONF_PLATFORM_STDIO_BUFFERED_SERIAL             1                               // set by library:platform
```

Comment:
this is a new options that replaces the older generic `Serial` object. I assumed that we want buffered instead of unbuffered.